tutorials/grids.ipynb

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
<p>"The parameter <code>info</code> decides which extra information is shown and the value of <code>alpha</code> determines the transparency of the grid.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/878/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>